Commands/WriterReplaceCommand.cs

```diff
@@ -0,0 +1,24 @@
+﻿using G1ANT.Language;
+
+namespace G1ANT.Addon.LibreOffice.Commands
+{
+    [Command(Name = "writer.replace", Tooltip = "Replaces text in the document")]
+    class WriterReplaceCommand : Command
+    {
+        public WriterReplaceCommand(AbstractScripter scripter) : base(scripter) { }
+
+        public class Arguments : CommandArguments
+        {
+            [Argument(Name = "word", Tooltip = "Enter the word to replace", Required = true)]
```

Comment:
word -> text. If somebody wants to lets say wants to replace `.` with `,` he will provide no words to be replaced.